spp_manual_eligibility/wizard/create_program_wizard.py

```diff
@@ -0,0 +1,46 @@
+# Part of OpenG2P. See LICENSE file for full copyright and licensing details.
+import logging
+
+from odoo import fields, models
+
+# from odoo.exceptions import ValidationError
+
+_logger = logging.getLogger(__name__)
+
+
+class G2PCreateNewProgramWiz(models.TransientModel):
+    _inherit = "g2p.program.create.wizard"
+
+    eligibility_kind = fields.Selection(selection_add=[("manual_eligibility", "Manual Eligibility")])
+
+    def _get_default_eligibility_manager_val(self, program_id):
+        is_manual_eligibility = False
+        eligibility_domain = self.eligibility_domain
+        eligibility_name = "Default"
+        for rec in self:
+            if rec.eligibility_kind == "manual_eligibility":
+                is_manual_eligibility = True
+                eligibility_domain = None
+                eligibility_name = "Manual Eligibility"
+
+        return {
+            "name": eligibility_name,
+            "program_id": program_id,
+            "admin_area_ids": self.admin_area_ids,
+            "eligibility_domain": eligibility_domain,
+            "is_manual_eligibility": is_manual_eligibility,
+        }
+
+    def _get_eligibility_manager(self, program_id):
+        val = {}
+        if self.eligibility_kind in ("default_eligibility", "manual_eligibility"):
+            # Add a new record to default eligibility manager model
+            default_eligibility_manager_val = self._get_default_eligibility_manager_val(program_id)
+            def_mgr = self.env["g2p.program_membership.manager.default"].create(default_eligibility_manager_val)
+
+            # Add a new record to eligibility manager parent model
+            eligibility_manager_val = self._get_eligibility_managers_val(program_id, def_mgr)
+            mgr = self.env["g2p.eligibility.manager"].create(eligibility_manager_val)
+
+            val = {"eligibility_managers": [(4, mgr.id)]}
+        return val
```

Comment:
use `super` here so that you will not overwrite other modules that is extending this method

kindly refer to the following files
- spp_eligibility_sql/wizard/create_program_wizard.py
- spp_eligibility_tags/wizard/create_program_wizard.py

Reply:
Done